pandas/tests/test_window.py

```diff
@@ -48,6 +48,12 @@ def win_types_special(request):
     return request.param
 
 
+@pytest.fixture(params=["sum", "mean", "median", "max", "min",
```

Comment:
ok this is fine; can you do a followup to use this fixture as much as possible in this file?